docs/design/expressions/member_access.md

```diff
@@ -841,6 +842,32 @@ alias X3 = (i32 as Factory).Make;
 alias X4 = i32.((i32 as Factory).Make);
 ```
 
+## Tuple indexing
+
+A tuple indexing expression is of the form:
+
+-   _expression_ `.` _integer-literal_
+-   _expression_ `->` _integer-literal_
+
+The _expression_ is required to be of tuple type, and _integer-literal_ is
+required to be less than the number of tuple elements. The result is the
+corresponding positional element of the tuple.
```

Comment:
Can the index be negative?

Reply:
No, because an *integer-literal* token is never negative.